### PR TITLE
Update Formatting model utils documentation

### PR DESCRIPTION
**Pull request by AleksSavelev**
The new version of powerbi-visuals-utils-formattingmodel@6.0.0 is available now

### CODE REVIEW

**Review by mberdugo — APPROVED** [review]

**Comment by mberdugo** [comment]
@AleksSavelev, when you say the method for using APIs without utils is described below, which section exactly? Also, is this for version 6 or 5.1? It still says 5.1.

**Comment by Court72** [comment]
@mberdugo

Thanks for reviewing the proposed changes.

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by AleksSavelev** [comment]
@mberdugo 

We have 2 different ways to implement the new format pane. The first one is described on the 'utils-formatting-model' page and the second one is on the 'format-pane'. So, 'using APIs without utils is described below' means that this page is related to the second way of developing the format pane.

Regarding the versions. The 5.1+ version is related to our `powerbi-visuals-api ` package, but I'm talking about the new version of `powerbi-visuals-utils-formattingmodel`

**Comment by mberdugo** [comment]
#sign-off

**Review by mberdugo — APPROVED** [review]

File: powerbi-docs/developer/visuals/format-pane.md [hunk 1]
```diff
@@ -17,15 +17,18 @@ Starting from API version 5.1, developers can create visuals that use the [new P
...
 The `getFormattingModel` returns a `FormattingModel` that defines how the visual's formatting and analytics pane look.
 
 In addition to all the old formatting pane capabilities, the [new formatting model](#formatting-model) supports new format pane capabilities, new properties and new hierarchies.
 
 :::image type="content" source="media/format-pane/formatting-model.png" alt-text="Screenshot of the new formatting pane.":::
 
-To upgrade to API version 5.1+, set the `apiVersion` in your *pbiviz.json* file to `5.1` or later.
+To upgrade to API version 5.1+, set the `apiVersion` in your *pbiviz.json* file to `5.1` or later and do **one** of the following:
+
+* Use [formattingmodel util](./utils-formatting-model.md). **Recommended**
```
Comment (Court72) [inline powerbi-docs/developer/visuals/format-pane.md:30]:
```suggestion
* Use [formattingmodel util](./utils-formatting-model.md). (**Recommended**)
```

File: powerbi-docs/developer/visuals/utils-formatting-model.md [hunk 2]
```diff
@@ -70,46 +72,85 @@ This example declares a new formatting settings model:
...
 
 ## Formatting settings card
 
-A *formatting settings card* specifies a formatting card in the formatting or analytics pane. A formatting settings card can contain multiple formatting slices, containers, and properties.
-Adding slices to a formatting settings card puts all of these slices into one formatting group.
+A *formatting settings card* specifies a formatting card in the formatting or analytics pane. A formatting settings card can contain multiple formatting slices, containers, groups, and properties.
+
+Adding slices to a formatting settings card puts all of these slices into one formatting card.
+
+Cards, Slices and Groups can be hidden dynamically by setting the `visible` parameter to *false* (*true* by default)
```
Comment (Court72) [inline powerbi-docs/developer/visuals/utils-formatting-model.md:94]:
```suggestion
Cards, Slices, and Groups can be hidden dynamically by setting the `visible` parameter to *false* (*true* by default).
```

**Comment by Court72** [comment]
Merging now.

**Pull request merged**